apps/portal/src/content/docs/components/drawer.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Drawer
+description: Drawer component with stacking
+---
+
+The Drawer component works similar to a Sheet component, but allows animated stacking as well.
+
+import { DocsPage } from "@/components/docs-page";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<Drawer.Root direction='right'>
+    <Drawer.Trigger><Button>Open Drawer</Button></Drawer.Trigger>
+    <Drawer.Content>
+      <Drawer.Header>
+        <Drawer.Title>Run Step</Drawer.Title>
+        <Drawer.Description>
```

Comment:
This composition looks wrong here. The Header shouldn't contain everything, only the title and description and the description should not contain all logic, just a description for a11y reasons. The body of the drawer should be a sibling to the Header/Footer.


```typescript tsx
<Drawer.Root>
  <Drawer.Trigger>Open it</Drawer.Trigger>

  <Drawer.Content>
    <Drawer.Header>
      <Drawer.Title>My drawer</Drawer>
      <Drawer.Description>My drawer is cool</Drawer.Description>
    </Drawer.Header>

    <p>This is the content of the drawer</p>

    <Drawer.Footer>
      <p>I'm in the footer</p>
    </Drawer.Footer>
  </Drawer.Content>
</Drawer.Root>
```